### DISCUSSION TITLE
Direction button not working on mobile

### DISCUSSION THREAD

**Discussion by willinrecovery** [code4recovery/12-step-meeting-list#1369] [Q&A]
So we have set up the maps for meetings and the maps themselves are showing up correctly on both desktop and mobile.

However, while the Directions button is working fine on the desktop (its seems to open a Google page) but it doesn't seem to work on mobile. I am just getting a blank screen.

Note: I am using the Arc browser (which is based on Chrome).  It does seem to work on Chrome itself, however. In Chrome it seems to open the Maps app on my iPhone.

![IMG_5618](https://github.com/code4recovery/12-step-meeting-list/assets/159723102/3c8ca03c-cb9d-4572-b258-e3d9069ed435)

**Comment by joshreisner** [comment]
ok thanks for the report @willinrecovery - i've created an issue https://github.com/code4recovery/tsml-ui/issues/365

**Reply by joshreisner** [reply]
@willinrecovery one other question - do you have the apple maps app installed? 

we got another issue this week that might be related: https://github.com/code4recovery/12-step-meeting-list/discussions/1378

**Reply by willinrecovery** [reply]
@joshreisner Yes, I use Apple maps for maps on my phone.

But Chrome does successfully bring the Apple Map app up.  Arc (a Chrome-based browser) does not, it just brings up the blank screen I showed above.
